src/common/constants.js

```diff
@@ -126,12 +130,13 @@ export const STUDY_SHORT_NAMES = [
   'Congenital Diaphragmatic Hernia',
 ];
 
+const protocol = window.location.protocol;
+const host = window.location.host;
+
```

Comment:
const URI = `${protocol}://${host}`

not required, but it was more what I meant. But still better now.